src/camera/Camera.zig

```diff
@@ -0,0 +1,19 @@
+const LinuxCamera = @import("linux/LinuxCamera.zig").LinuxCamera;
+const MacOSCamera = @import("macos/MacOSCamera.zig").MacOSCamera;
+
+pub const Camera = union(enum) {
```

Comment:
Simple cross-platform camera abstraction. Platform specific stuff will be handled in `LinuxCamera` and `MacOSCamera`.